lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb

```diff
@@ -643,6 +644,76 @@ def scrape_passwords(provider, method, args)
     return true
   end
 
+  def report_smb_cred(credential_core)
+    # Assemble the options hash for creating the Metasploit::Credential::Login object
+    login_data = {
+      core: credential_core,
+      status: Metasploit::Model::Login::Status::UNTRIED,
+      address: ::Rex::Socket.getaddress(client.sock.peerhost, true),
+      port: 445,
+      service_name: 'smb',
+      protocol: 'tcp',
+      workspace_id: shell.framework.db.workspace.id
+    }
+
+    shell.framework.db.create_credential_login(login_data)
+  end
+
+  def create_cred(domain, credential_data)
```

Comment:
Domain can be blank and I agree this would be a better signature. Also FYI edited your comment to fix a typo on the `blank` part. Will see if I can't update this and the associated calls.

Reply:
Should be fixed in the upcoming commit.